installer.iss

```diff
@@ -11,6 +10,14 @@
 #define DotNetInstallerTitle "Microsoft .NET 6 Desktop Runtime"
 
 #define TargetFramework "net6.0-windows"
+#define ReleasePath "RetroBar\bin\Release\" + TargetFramework
+#define LanguagePath "RetroBar\Languages"
+
+#define Major
+#define Minor
+#define Revision
+#define Build
+#define RetroBarVersion GetVersionComponents(ReleasePath + "\publish-x64\" + RetroBarExeName, Major, Minor, Revision, Build), Str(Major) + "." + Str(Minor) + "." + Str(Revision)
 
 [Setup]
 AppId={{574527FE-00A4-4F85-92AD-B4B8B4077D73}
```

Comment:
Should this be "`{{`"?

Reply:
Yes, the extra `{` is used as an escape character in this instance.